tunnistamo/tests/test_locale_middleware.py

```diff
@@ -16,10 +17,17 @@ def test_language_is_determined_by_django_locale_middleware_if_ui_locales_provid
     assert f'<html lang="{expected_language}">' in response.rendered_content
 
 
+NON_DEFAULT_LANGUAGE_CODE = list(filter(lambda lang: lang[0] != settings.LANGUAGE_CODE, settings.LANGUAGES))[0][0]
+
+
 @pytest.mark.django_db
-@pytest.mark.parametrize('ui_locales', ['fi', 'sv', 'en'])
-def test_language_is_determined_from_ui_locales_query_parameter(ui_locales, client, use_translations):
+@pytest.mark.parametrize('ui_locales,expected_lang', [
+    (f'{settings.LANGUAGE_CODE} {NON_DEFAULT_LANGUAGE_CODE}', settings.LANGUAGE_CODE),
```

Comment:
Could be a good idea to include a code with the region too? e.g. `fi-FI` or `en-US`

Reply:
Added. It's a non-existing specifier, but I guess Django doesn't care (and I guess it shouldn't be picky, because more specifiers can appear).

Reply:
I thought that it's not that important to test the `get_supported_language_variant` functionality that thoroughly here. Just trust that it does its job.